Packs/Syslog/.secrets-ignore

```diff
@@ -1,3 +1,5 @@
 SysLogHandler
 127.0.0.1
-www.eizelulz.com
\ No newline at end of file
+www.eizelulz.com
+aHR0cHM6Ly93d3cuZXhhbXBsZS5jb20vdGVzd
```

Comment:
Are we sure that those are secrets??

Reply:
I got alart at start but now I don't so I'm removing it from there :)